### PR TITLE
Docs-calico-hardway

### PR DESCRIPTION
**Pull request by bmckercher123**
## Description

Per https://github.com/projectcalico/calico/issues/4150, update Calico the hard way doc 
with note that it is not meant for production installs.

- Merge to nightly and cherrypick to 3.17 (OS only)

### CODE REVIEW

File: getting-started/kubernetes/hardway/overview.md [hunk 1]
```diff
@@ -4,16 +4,17 @@ description: A tutorial for installing Calico the hard way.
...
 ---
 
-This tutorial walks you through setting up {{site.prodname}} the hard way.
+### About this tutorial
 
- - If you are looking to get up and running quickly with {{site.prodname}}, to try things out, check out our [quickstart guide]({{site.baseurl}}/getting-started/kubernetes/quickstart).
- - If you are looking for the most direct path to a production-ready {{site.prodname}} install, check out our [install guides]({{site.baseurl}}/getting-started/kubernetes/self-managed-onprem/onpremises).
-
-{{site.prodname}} the hard way is optimized for learning about how {{site.prodname}} works and what the other guides do “under the hood.”
+The "Hard Way" is a tutorial for learning how Kubernetes and {{site.prodname}} components fit together. It is a manifest-based install, and is optimized for learning what happens “under the hood.” **It is not intended for production installs** (which is an operator-based).
```
Comment (fasaxc) [inline getting-started/kubernetes/hardway/overview.md:9]: Mixed ascii and unicode quotes in this para.

"which is an operator based": "which" doesn't refer back to anything, and, "an operator-based _what_"? 

In open source, the manifest-based installs are production ready too; just not this particular one. (In addition, the operator is not rolled out to all scenarios yet.)

File: getting-started/kubernetes/hardway/overview.md [hunk 2]
```diff
@@ -4,16 +4,17 @@ description: A tutorial for installing Calico the hard way.
...
 
 The name “{{site.prodname}} the hard way” is inspired by {% include open-new-window.html text='Kubernetes the hard way' url='https://github.com/kelseyhightower/kubernetes-the-hard-way' %} by Kelsey Hightower.
 
-## Target Audience
+ - To get up and running quickly with {{site.prodname}}, and try things out, see the [quickstart guide]({{site.baseurl}}/getting-started/kubernetes/quickstart).
+ - For the most direct path to a production-ready {{site.prodname}} install, see our [install guides]({{site.baseurl}}/getting-started/kubernetes/self-managed-onprem/onpremises).
```
Comment (fasaxc) [inline getting-started/kubernetes/hardway/overview.md:14]: Should this link to onprem only?

**Review by fasaxc — CHANGES_REQUESTED** [review]

File: getting-started/kubernetes/hardway/overview.md (hunk 2 above)
Comment (bmckercher123) [reply getting-started/kubernetes/hardway/overview.md]: Yeah, I wondered that as well. Let's drop it and assume users can discern quickstart from hard way.

File: getting-started/kubernetes/hardway/overview.md (hunk 1 above)
Comment (bmckercher123) [reply getting-started/kubernetes/hardway/overview.md]: Argh! Rewrite includes your suggestions.

**Review by fasaxc — APPROVED** [review]
Reads clearly now. LGTM

**Pull request merged**